src/main/ts/core/NpmTags.ts

```diff
@@ -3,25 +3,33 @@ import * as Git from '../utils/Git';
 import * as PromiseUtils from '../utils/PromiseUtils';
 import * as ArrayUtils from '../utils/ArrayUtils';
 import { BranchState, getReleaseBranchName, versionFromReleaseBranch } from './BranchLogic';
-import * as Version from './Version';
-
-const normalize = (branchName: string): string =>
-  branchName.replace(/[^\w.]+/g, '-');
-
-export const pickTags = async (branchName: string, branchState: BranchState, getBranches: () => Promise<string[]>): Promise<[string, string?]> => {
-  const mainTag = normalize(branchName);
-
-  if (branchState !== BranchState.ReleaseReady) {
-    return [ mainTag ];
-  }
+import { compareMajorMinorVersions, majorMinorVersionToString, Version } from './Version';
 
+const isLatestReleaseBranch = async (getBranches: () => Promise<string[]>, branchName: string): Promise<boolean> => {
```

Comment:
Why not just pass the branches in directly?

Reply:
Not all code paths require the list of branches, so we'd be doing an HTTP call unnecessarily.